template/macro/2015/layout.html

```diff
@@ -50,38 +44,37 @@
 {% set video = video[0] %}
 {% set picture = [video.elements[0].assets[0]] %}
 {% set stillImageUrl = picture.file %}
+    {% for media_asset in video.elements[0].assets if '480' == media_asset.typeData.width or '460' == media_asset.typeData.width %}
+        {% if picture.append(media_asset) %}{% endif %}
```

Comment:
This might be appearing because I removed this macro for a while and then reverted it back. But it has had no change on my side, it is still the code that has been provided and it is a macro i am not using at all. You can validate this by cross-checking with the original template/macro/layout.html.